packages/harmony/src/components/layout/Box/Box.tsx

```diff
@@ -61,6 +62,8 @@ export const Box = styled.div<BoxProps>(
       marginLeft: marginL && spacing[marginL],
       marginRight: marginR && spacing[marginR],
       marginBottom: marginB && spacing[marginB],
+      backgroundColor:
```

Comment:
realizing its a common enough pattern to apply background color to a flex/box, not just a paper. @dharit-tan also cause i know you've wanted this too.